2011-12-28 12:01:48,132 [Thread-10] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractSavePersistentMethod.doInvokeInternal(AbstractSavePersistentMethod.java:182)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractDynamicPersistentMethod.invoke(AbstractDynamicPersistentMethod.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoCachedMethodSite.invoke(PojoMetaMethodSite.java:189)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:53)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:124)
	at org.codehaus.groovy.grails.orm.hibernate.HibernateGormInstanceApi.save(HibernateGormEnhancer.groovy:882)
	at edu.mcw.User.save(User.groovy)
	at edu.mcw.User$save.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:928)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at org.codehaus.groovy.grails.commons.DefaultGrailsBootstrapClass.callInit(DefaultGrailsBootstrapClass.java:60)
	at org.codehaus.groovy.grails.web.context.GrailsConfigUtils.executeGrailsBootstraps(GrailsConfigUtils.java:74)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:105)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:01:48,154 [Thread-10] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:01:48,154 [Thread-10] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:01:48,155 [Thread-10] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:01:48,318 [Thread-10] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:118)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
2011-12-28 12:01:48,318 [Thread-10] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:01:48,318 [Thread-10] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: grails.validation.ValidationException: Validation Error(s) occurred during save():
- Field error in object 'edu.mcw.User' on field 'hashed_password': rejected value [null]; codes [edu.mcw.User.hashed_password.nullable.error.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.error.hashed_password,edu.mcw.User.hashed_password.nullable.error.java.lang.String,edu.mcw.User.hashed_password.nullable.error,user.hashed_password.nullable.error.edu.mcw.User.hashed_password,user.hashed_password.nullable.error.hashed_password,user.hashed_password.nullable.error.java.lang.String,user.hashed_password.nullable.error,edu.mcw.User.hashed_password.nullable.edu.mcw.User.hashed_password,edu.mcw.User.hashed_password.nullable.hashed_password,edu.mcw.User.hashed_password.nullable.java.lang.String,edu.mcw.User.hashed_password.nullable,user.hashed_password.nullable.edu.mcw.User.hashed_password,user.hashed_password.nullable.hashed_password,user.hashed_password.nullable.java.lang.String,user.hashed_password.nullable,nullable.edu.mcw.User.hashed_password,nullable.hashed_password,nullable.java.lang.String,nullable]; arguments [hashed_password,class edu.mcw.User]; default message [Property [{0}] of class [{1}] cannot be null]

	at BootStrap$_closure1.doCall(BootStrap.groovy:7)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
2011-12-28 12:07:00,706 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:97)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:928)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at org.codehaus.groovy.grails.commons.DefaultGrailsBootstrapClass.callInit(DefaultGrailsBootstrapClass.java:60)
	at org.codehaus.groovy.grails.web.context.GrailsConfigUtils.executeGrailsBootstraps(GrailsConfigUtils.java:74)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:105)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:55)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:51)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	... 23 more
2011-12-28 12:07:00,806 [Thread-9] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:55)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:51)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:928)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at org.codehaus.groovy.grails.commons.DefaultGrailsBootstrapClass.callInit(DefaultGrailsBootstrapClass.java:60)
	at org.codehaus.groovy.grails.web.context.GrailsConfigUtils.executeGrailsBootstraps(GrailsConfigUtils.java:74)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:105)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:00,842 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:00,985 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:01,086 [Thread-9] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:01,119 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:01,479 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:118)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:01,597 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:01,730 [Thread-9] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:01,754 [Thread-9] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.codehaus.groovy.runtime.InvokerInvocationException: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
Caused by: groovy.lang.MissingMethodException: No signature of method: edu.mcw.User.errors() is applicable for argument types: () values: []
Possible solutions: getErrors(), setErrors(org.springframework.validation.Errors), hasErrors(), every(), equals(java.lang.Object), exists(java.io.Serializable)
	at BootStrap$_closure1.doCall(BootStrap.groovy:18)
	... 8 more
2011-12-28 12:07:02,286 [Thread-16] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.BeanCreationNotAllowedException: Error creating bean with name 'scaffoldedActionMap': Singleton bean creation not allowed while the singletons of this factory are in destruction (Do not request a bean from a BeanFactory in a destroy method implementation!)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:212)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:291)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1093)
	at org.springframework.beans.factory.BeanFactory$getBean.call(Unknown Source)
	at org.codehaus.groovy.grails.plugins.scaffolding.ScaffoldingGrailsPlugin.configureScaffoldingController(ScaffoldingGrailsPlugin.groovy:101)
	at org.codehaus.groovy.grails.plugins.scaffolding.ScaffoldingGrailsPlugin$configureScaffoldingController.callStatic(Unknown Source)
	at org.codehaus.groovy.grails.plugins.scaffolding.ScaffoldingGrailsPlugin.configureScaffolding(ScaffoldingGrailsPlugin.groovy:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:361)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:704)
	at groovy.lang.GroovyObjectSupport.invokeMethod(GroovyObjectSupport.java:44)
	at groovy.lang.MetaClassImpl.invokeMethodOnGroovyObject(MetaClassImpl.java:1119)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1013)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:66)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:46)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:145)
	at org.codehaus.groovy.grails.plugins.scaffolding.ScaffoldingGrailsPlugin$_closure2_closure10.doCall(ScaffoldingGrailsPlugin.groovy:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSite.invoke(PogoMetaMethodSite.java:226)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:52)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:46)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:141)
	at org.codehaus.groovy.grails.plugins.scaffolding.ScaffoldingGrailsPlugin$_closure2_closure10.doCall(ScaffoldingGrailsPlugin.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at groovy.lang.Closure.run(Closure.java:488)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:02,287 [Thread-16] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.BeanCreationNotAllowedException: Error creating bean with name 'scaffoldedActionMap': Singleton bean creation not allowed while the singletons of this factory are in destruction (Do not request a bean from a BeanFactory in a destroy method implementation!)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:02,287 [Thread-16] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.BeanCreationNotAllowedException: Error creating bean with name 'scaffoldedActionMap': Singleton bean creation not allowed while the singletons of this factory are in destruction (Do not request a bean from a BeanFactory in a destroy method implementation!)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:07:02,287 [Thread-16] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.BeanCreationNotAllowedException: Error creating bean with name 'scaffoldedActionMap': Singleton bean creation not allowed while the singletons of this factory are in destruction (Do not request a bean from a BeanFactory in a destroy method implementation!)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:17:41,614 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:83)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	at org.springframework.orm.hibernate3.HibernateAccessor.convertJdbcAccessException(HibernateAccessor.java:424)
	at org.springframework.orm.hibernate3.HibernateAccessor.convertHibernateAccessException(HibernateAccessor.java:410)
	at org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:411)
	at org.springframework.orm.hibernate3.HibernateTemplate.execute(HibernateTemplate.java:339)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.SavePersistentMethod.performSave(SavePersistentMethod.java:56)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractSavePersistentMethod.doInvokeInternal(AbstractSavePersistentMethod.java:207)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractDynamicPersistentMethod.invoke(AbstractDynamicPersistentMethod.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoCachedMethodSite.invoke(PojoMetaMethodSite.java:189)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:53)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:55)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:124)
	at org.codehaus.groovy.grails.orm.hibernate.HibernateGormInstanceApi.save(HibernateGormEnhancer.groovy:882)
	at edu.mcw.Project.save(Project.groovy)
	at edu.mcw.Project$save.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:928)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at org.codehaus.groovy.grails.commons.DefaultGrailsBootstrapClass.callInit(DefaultGrailsBootstrapClass.java:60)
	at org.codehaus.groovy.grails.web.context.GrailsConfigUtils.executeGrailsBootstraps(GrailsConfigUtils.java:74)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:105)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.hibernate.id.IdentityGenerator$GetGeneratedKeysDelegate.executeAndExtract(IdentityGenerator.java:94)
	at org.hibernate.id.insert.AbstractReturningDelegate.performInsert(AbstractReturningDelegate.java:57)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2345)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2852)
	at org.hibernate.action.EntityIdentityInsertAction.execute(EntityIdentityInsertAction.java:71)
	at org.hibernate.engine.ActionQueue.execute(ActionQueue.java:273)
	at org.codehaus.groovy.grails.orm.hibernate.support.ClosureEventTriggeringInterceptor.performSaveOrReplicate(ClosureEventTriggeringInterceptor.java:249)
	at org.hibernate.event.def.AbstractSaveEventListener.performSave(AbstractSaveEventListener.java:203)
	at org.hibernate.event.def.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:129)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:210)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:195)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.performSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:117)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:93)
	at org.codehaus.groovy.grails.orm.hibernate.support.ClosureEventTriggeringInterceptor.onSaveOrUpdate(ClosureEventTriggeringInterceptor.java:106)
	at org.hibernate.impl.SessionImpl.fireSaveOrUpdate(SessionImpl.java:685)
	at org.hibernate.impl.SessionImpl.saveOrUpdate(SessionImpl.java:677)
	at org.hibernate.impl.SessionImpl.saveOrUpdate(SessionImpl.java:673)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.SavePersistentMethod$1.doInHibernate(SavePersistentMethod.java:58)
	at org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:406)
	... 50 more
2011-12-28 12:17:41,618 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.hibernate.id.IdentityGenerator$GetGeneratedKeysDelegate.executeAndExtract(IdentityGenerator.java:94)
	at org.hibernate.id.insert.AbstractReturningDelegate.performInsert(AbstractReturningDelegate.java:57)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2345)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:2852)
	at org.hibernate.action.EntityIdentityInsertAction.execute(EntityIdentityInsertAction.java:71)
	at org.hibernate.engine.ActionQueue.execute(ActionQueue.java:273)
	at org.codehaus.groovy.grails.orm.hibernate.support.ClosureEventTriggeringInterceptor.performSaveOrReplicate(ClosureEventTriggeringInterceptor.java:249)
	at org.hibernate.event.def.AbstractSaveEventListener.performSave(AbstractSaveEventListener.java:203)
	at org.hibernate.event.def.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:129)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:210)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:195)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.performSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:117)
	at org.hibernate.event.def.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:93)
	at org.codehaus.groovy.grails.orm.hibernate.support.ClosureEventTriggeringInterceptor.onSaveOrUpdate(ClosureEventTriggeringInterceptor.java:106)
	at org.hibernate.impl.SessionImpl.fireSaveOrUpdate(SessionImpl.java:685)
	at org.hibernate.impl.SessionImpl.saveOrUpdate(SessionImpl.java:677)
	at org.hibernate.impl.SessionImpl.saveOrUpdate(SessionImpl.java:673)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.SavePersistentMethod$1.doInHibernate(SavePersistentMethod.java:58)
	at org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:406)
	at org.springframework.orm.hibernate3.HibernateTemplate.execute(HibernateTemplate.java:339)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.SavePersistentMethod.performSave(SavePersistentMethod.java:56)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractSavePersistentMethod.doInvokeInternal(AbstractSavePersistentMethod.java:207)
	at org.codehaus.groovy.grails.orm.hibernate.metaclass.AbstractDynamicPersistentMethod.invoke(AbstractDynamicPersistentMethod.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoCachedMethodSite.invoke(PojoMetaMethodSite.java:189)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:53)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:55)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:124)
	at org.codehaus.groovy.grails.orm.hibernate.HibernateGormInstanceApi.save(HibernateGormEnhancer.groovy:882)
	at edu.mcw.Project.save(Project.groovy)
	at edu.mcw.Project$save.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1231)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:233)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1054)
	at groovy.lang.ExpandoMetaClass.invokeMethod(ExpandoMetaClass.java:1110)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:928)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:884)
	at groovy.lang.Closure.call(Closure.java:410)
	at groovy.lang.Closure.call(Closure.java:404)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at org.codehaus.groovy.grails.commons.DefaultGrailsBootstrapClass.callInit(DefaultGrailsBootstrapClass.java:60)
	at org.codehaus.groovy.grails.web.context.GrailsConfigUtils.executeGrailsBootstraps(GrailsConfigUtils.java:74)
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:105)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:17:41,619 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
2011-12-28 12:17:41,620 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
2011-12-28 12:17:41,620 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:17:41,620 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
2011-12-28 12:17:41,792 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.codehaus.groovy.grails.web.context.GrailsContextLoader.initWebApplicationContext(GrailsContextLoader.java:118)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
	at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
2011-12-28 12:17:41,793 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
2011-12-28 12:17:41,793 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
2011-12-28 12:17:41,793 [Thread-11] ERROR StackTrace  - Full Stack Trace:
org.springframework.beans.factory.access.BootstrapException: Error executing bootstraps; nested exception is org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: could not insert: [edu.mcw.Project]; uncategorized SQLException for SQL [insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?)]; SQL state [90006]; error code [90006]; NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]; nested exception is org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at BootStrap$_closure1.doCall(BootStrap.groovy:31)
	at grails.util.Environment.evaluateEnvironmentSpecificBlock(Environment.java:290)
	at grails.util.Environment.executeForEnvironment(Environment.java:283)
	at grails.util.Environment.executeForCurrentEnvironment(Environment.java:259)
	... 5 more
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "CODE_ID"; SQL statement:
insert into project (id, version, active, code_id, description, name) values (null, ?, ?, ?, ?, ?) [90006-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:294)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:630)
	at org.h2.command.dml.Insert.insertRows(Insert.java:122)
	at org.h2.command.dml.Insert.update(Insert.java:86)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:143)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:129)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	at org.apache.commons.dbcp.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:105)
	... 9 more
